Mathlib/Algebra/MvPolynomial/Degrees.lean

```diff
@@ -291,7 +291,7 @@ theorem degreeOf_mul_X_ne {i j : σ} (f : MvPolynomial σ R) (h : i ≠ j) :
   simp only [Finsupp.single, Nat.one_ne_zero, add_right_eq_self, addRightEmbedding_apply, coe_mk,
     Pi.add_apply, comp_apply, ite_eq_right_iff, Finsupp.coe_add, Pi.single_eq_of_ne h]
 
-theorem degreeOf_mul_X_eq (j : σ) (f : MvPolynomial σ R) :
+theorem degreeOf_mul_X_le (j : σ) (f : MvPolynomial σ R) :
```

Comment:
Oh wait, now I notice `degreeOf_mul_X_ne`, and it refers to the indices being the same. I think better names might be `degreeOf_mul_X_of_ne` and `degreeOf_mul_X_self`. What do you think?

Reply:
also did a little bit of golfing.